tests/codeception/functional/MegaMenuCest.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+use Drupal\Core\Url;
+
+/**
+ * Class MegaMenuCest.
+ *
+ * @group install
+ */
+class MegaMenuCest {
+
+  /**
+   * Every main menu item should not error.
+   */
+  public function testMegaMenu(FunctionalTester $I) {
+
+    $I->logInWithRole('administrator');
+    $top_level = $I->createEntity([
+      'title' => 'Top Level Page',
+      'type' => 'hs_basic_page',
+    ]);
+
+    $I->amOnPage($top_level->toUrl('edit-form')->toString());
+    $I->click('.menu-link-form summary');
+    $I->checkOption('Provide a menu link');
+    $I->fillField('Menu link title', 'Top Level Page');
+    $I->scrollTo(['css' => '.form-submit']);
+    $I->click('Save');
+
+    $second_level = $I->createEntity([
+      'title' => 'Second Level Page',
+      'type' => 'hs_basic_page',
+    ]);
+    $I->amOnPage($second_level->toUrl('edit-form')->toString());
+    $I->click('.menu-link-form summary');
+    $I->checkOption('Provide a menu link');
+    $I->fillField('Menu link title', 'Second Level Page');
+    $I->scrollTo(['css' => '.form-item-menu-menu-parent'], 0, -100);
+    $I->selectOption('Parent link', '-- Top Level Page');
+    $I->wait(2);
+    $I->click('Show row weights');
+    $I->scrollTo(['css' => '.form-submit']);
+    $I->click('Save');
+
+    $I->amOnPage('/admin/structure/menu/manage/main');
+    $I->see('Top Level Page');
+    $I->see('Second Level Page');
+
+    $I->amOnPage('/admin/config/site-options');
+    $I->see('Enable New Mega Menu');
+
+    $this->megaMenuEnabled = (bool) $I->grabAttributeFrom('[name="field_en_mega_menu[value]"]', 'checked');
```

Comment:
You can use the field label.

Reply:
Can't use the field label for this - see here: https://codeception.com/docs/modules/WebDriver#grabAttributeFrom, it needs CSS or XPath.

Reply:
ahh yeah. :+1: